CHANGES.md

```diff
@@ -0,0 +1,6303 @@
+
+### Other changes
+
+- [#2709](https://github.com/gap-system/gap/pull/2709) Rename `MultRowVector` to `MultVector` (the old name is still supported, but marked as obsolete)
+- [#2729](https://github.com/gap-system/gap/pull/2729) Rename `QuaternionGroup` to `DicyclicGroup`, document `IsDihedralGroup` and `IsQuaternionGroup`
+- [#3010](https://github.com/gap-system/gap/pull/3010) Read `lib/transatl.g` before any `gap.ini` file, when GAP is loaded
+- [#3406](https://github.com/gap-system/gap/pull/3406) Remove RXVT-shell support for Windows
+- [#3480](https://github.com/gap-system/gap/pull/3480) `BuildPackages.sh` now executes `make clean` before full build
+
+### New packages redistributed with GAP
+- [CddInterface](https://homalg-project.github.io/CddInterface/) GAP interface to cdd, by Kamal Saleh
+- [DifSets](https://dylanpeifer.github.io/difsets/)  Enumeration of the difference sets (up to equivalence) in 
+ groups,  by Dylan Peifer
+- [ferret](https://gap-packages.github.io/ferret/) C++ reimplementation of Jeffery Leon’s Partition Backtrack framework for solving problems in permutation groups, by Christopher Jefferson
+- [images](https://gap-packages.github.io/images/) Finding minimal and canonical images in permutation groups, by Christopher Jefferson, Markus Pfeiffer, Rebecca Waldecker, Eliza Jonauskyte
+- [NCOnvex](https://homalg-project.github.io/NConvex/) Polyhedral constructions and computations for cones, polyhedrons, polytopes and fans, by Kamal Saleh, Sebastian Gutsche
+- [NoCK](https://pjastr.github.io/NoCK/) Computation of Tolzanos’s obstruction for compact Clifford-Klein forms, by Maciej Bocheński, Piotr Jastrzębski, Anna Szczepkowska, Aleksy Tralle, Artur Woike
+
+
+## GAP 4.10.2 (June 2019)
+
+### Changes in the core GAP system introduced in GAP 4.10.21
+
+Improvements in the experimental way to allow 3rd party code to link
+GAP as a library:
+
+  - Add `GAP_AssignGlobalVariable` and
+    `GAP_IsNameOfWritableGlobalVariable` to the **libGAP** API
+    ([#3438](https://github.com/gap-system/gap/pull/3438)).
+
+Fixes in the experimental support for using the **Julia** garbage
+collector:
+
+  - Fix of a problem where the Julia GC during a partial sweep frees
+    some, but not all objects of an unreachable data structure, and also
+    may erroneously try to mark the deallocated objects
+    ([#3412](https://github.com/gap-system/gap/pull/3412)).
+
+  - Fix stack scanning for the Julia GC when GAP is used as a library
+    ([#3432](https://github.com/gap-system/gap/pull/3432)).
+
+Fixed bugs that could lead to crashes:
+
+  - Fix a bug in `TransformationListList` (Reference:
+    TransformationListList for a source and destination) which could
+    cause a crash
+    ([#3463](https://github.com/gap-system/gap/pull/3463)).
+
+Fixed bugs that could lead to incorrect results:
+
+  - Fix a bug in `ClassPositionsOfLowerCentralSeries` (Reference:
+    ClassPositionsOfLowerCentralSeries). \[Reported by Frieder Ladisch\]
```

Comment:
These `\[ ... \]` entries are annoying. Perhaps we can replace them by `( ... )` via a regex, or even just drop the brackets. So one of these (but applied globally):
```suggestion
    ClassPositionsOfLowerCentralSeries). (Reported by Frieder Ladisch)
```
```suggestion
    ClassPositionsOfLowerCentralSeries). Reported by Frieder Ladisch
```